rocketpy/utilities.py

```diff
@@ -229,52 +229,51 @@ def fin_flutter_analysis(
     ------
     None
     """
+    found_fin = False
 
-    # First, we need identify if there is at least a fin set in the rocket
-    for aero_surface in flight.rocket.aerodynamic_surfaces:
```

Comment:
Just to make reviewer's life easier: this was the problematic code line. The aerodynamic_surfaces is no longer a list of surfaces, instead it is a list of components;